spec/support/subscriptions_helpers.rb

```diff
@@ -29,5 +29,16 @@ def prepare_subscription_permissions(permissions, provider = 'MMT_2')
 
       [group_response['concept_id'], permission_response['concept_id']]
     end
+
+    def ingest_granules(collection_entry_title, count, provider = 'MMT_2')
```

Comment:
all the other ingest helper methods use keyword arguments, you might want to make this one consistent.
Since all the other helpers for ingest are in the drafts_helpers and now you have a separate one for granules for subscriptions, maybe we can consolidate all the ingest related helper methods to an ingest_helpers file?